test/api/test_dataset_api.py

```diff
@@ -0,0 +1,29 @@
+import pytest
```

Comment:
Small thing, but the file is called "dataset_api.py", as opposed to "datasets_api.py".

Reply:
updated, and for model